### ISSUE TITLE
Correção dos richpictures existentes

### ISSUE THREAD

**Issue by williamelias** [williamelias/Req-01-2019-medium#9]
- [ ] Padronizar richpictures (seguir o modelo do @pedro-prp )

- [ ]  Definir enumeração dos fluxos, quando necessário

- [ ] Definir bem a semântica envolvida, acrescentar nomes às setas

**Comment by gabrielfilipe7unb** [comment]
Foi feito na minha todas as modificações exigidas, inclusive já mostradas ao professor Maurício.

**Comment by gabrielfilipe7unb** [comment]
Creio que " Padronizar richpictures (seguir o modelo do @pedro-prp ) " não seja o ideal, mas sim, padronizar de acordo com a identidade do software.

**Comment by williamelias** [comment]
Concordo, @gabrielfilipe7unb , me expressei mau. Eu estava me referindo as setas e utilização da semântica.

**Comment by williamelias** [comment]
adicionado um novo critério referente a identidade visual na confecção dos richpictures @gabrielfilipe7unb

**Comment by pedro-prp** [comment]
Feito
